edb/testbase/server.py

```diff
@@ -382,7 +382,7 @@ def http_con_read_response(
     ) -> tuple[bytes, dict[str, str], int]:
         resp = http_con.getresponse()
         resp_body = resp.read()
-        resp_headers = {k.lower(): v.lower() for k, v in resp.getheaders()}
+        resp_headers = {k.lower(): v for k, v in resp.getheaders()}
```

Comment:
I think this will break some existing tests, but I'll fix those in a separate PR based on this to keep the noise down.

Reply:
Yeah, it's a good idea to do that as a totally separate PR

Reply:
https://github.com/edgedb/edgedb/pull/5824

Reply:
I don't see this change in #5824.